frontend/src/testing/setup.ts

```diff
@@ -1,5 +1,6 @@
 import "@testing-library/jest-dom/vitest";
 import { cleanup, configure } from "@testing-library/react";
+import "blob-polyfill";
```

Comment:
Did you try the approach suggested in https://github.com/jsdom/jsdom/issues/2555#issuecomment-1864762292?

Reply:
Looks like that works as well, I've had to add a line to polyfill File as well, and I had to disable eslint for those two lines to allow the conversion to any (the types of the nodejs ones and the browser ones technically don't match I believe). Changed in [c81b854](https://github.com/kiesraad/abacus/pull/1406/commits/c81b8546ff6acbdab351232b3fccda2db862c992)